plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/render/impl/RenderJob.java

```diff
@@ -282,6 +314,10 @@ public void setBounds(ReferencedEnvelope bounds) {
         this.bounds = bounds;
     }
 
+    public void disposeRenderExecutor() {
```

Comment:
its not API  and its not called internally. What is this method for?

Reply:
I will revert this.